bool is_pot_fire

uint16 img_width
uint16 img_height

# the x and y pos (sub-pixel) of the single fire, opencv style coordinates
float32 img_x
float32 img_y
